packages/strapi-provider-email-nodemailer/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "strapi-provider-email-nodemailer",
+  "version": "3.0.0",
+  "description": "Nodemailer provider for Strapi 3",
+  "main": "./lib",
+  "directories": {
+    "lib": "./lib"
+  },
+  "dependencies": {
+    "dotenv": "8.1.0",
+    "lodash": "4.17.19",
+    "nodemailer": "6.3.0"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "release": "node -r dotenv/config node_modules/release-it/bin/release-it.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/yutikom/strapi-provider-email-nodemailer.git"
+  },
+  "keywords": [
+    "Strapi",
+    "Email",
+    "Provider",
+    "Strapi 3"
+  ],
+  "author": {
+    "name": "Yurii Tykhomyrov"
+  },
+  "contributors": [
```

Comment:
Need to add maintainers block: https://github.com/strapi/strapi/blob/dc0a91c7be8f7e515617459875fcb865c0f68300/packages/strapi-provider-email-amazon-ses/package.json#L20-L30 for an example